src/components/users-form.tsx

```diff
@@ -35,18 +37,40 @@ const UsersFormSchema = z.object({
 
 export function UsersForm({
   defaultValues,
+  id,
 }: {
   defaultValues?: z.infer<typeof UsersFormSchema>
+  id: string
```

Comment:
should make id optional for when we are not editing form

Reply:
Thanks for catching this! Addressed in https://github.com/cs301-itsa-ay2324/admin-ui-proxy/pull/5/commits/a8ec0eb3a363bb5890f19bf1a0954402ad5ba618